documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -160,14 +160,57 @@ include::{testDir}/example/DisplayNameGeneratorDemo.java[tags=user_guide]
   | | +-- For example, year -1 is not supported. [OK]
   | | '-- For example, year -4 is not supported. [OK]
   | '-- if it is zero() [OK]
-  '-- A year is a leap year... [OK]
-    +-- A year is a leap year if it is divisible by 4 but not by 100. [OK]
-    '-- A year is a leap year if it is one of the following years. [OK]
+  '-- A year is a leap year [OK]
+    +-- A year is a leap year, if it is divisible by 4 but not by 100. [OK]
+    '-- A year is a leap year, if it is one of the following years. [OK]
       +-- Year 2016 is a leap year. [OK]
       +-- Year 2020 is a leap year. [OK]
       '-- Year 2048 is a leap year. [OK]
 ```
 
+[[writing-tests-display-name-generator-extensions]]
+==== Extensions of Display Name Generators
+The class DisplayNameGenerator already supports the extensions of `ReplaceUnderscores`
+and `IndicativeSentences`.
+
+By using the annotation that extends `Standard`:
```

Comment:
❓ Did you mean this?

```suggestion
By using the annotation with a class that extends `Standard`:
```

Reply:
Yes, with this changes I'll update that and value for generator in the documentation